validation/index.js

```diff
@@ -27,19 +27,19 @@ function test(fileContent) {
     if (json.status == "PREPARATION") {
         // If status == PREPARATION, does before_login_hint exist?
         if (!json.before_login_hint || json.before_login_hint.trim() === "") {
-            throw new Error("Status is Preparation, but 'before_login_hint' is missing")
+            throw new Error("Status is PREPARATION, but 'before_login_hint' is missing")
         }
         // If status == PREPARATION, does markdown exist? (maybe even require screenshots?)
         if (markdown.trim() === "") {
-            throw new Error("Status is Preparation, but website content is missing")
+            throw new Error("Status is PREPARATION, but website content is missing")
         }
     } else if (json.status == "BROKEN") {
         // If status == BROKEN, does before_login_hint & after_login_hint exist?
         if (!json.before_login_hint || json.before_login_hint.trim() === "") {
-            throw new Error("Status is Broken, but 'before_login_hint' is missing")
+            throw new Error("Status is BROKEN, but 'before_login_hint' is missing")
         }
-        if (!json.after_login_hint || json.after_login_hint.trim() === "") {
-            throw new Error("Status is Broken, but 'after_login_hint' is missing")
+        if (markdown.trim() === "") {
+            throw new Error("Status is BROKEN, but website content is missing")
```

Comment:
We do need the after_login_hint for broken providers as well. As we don't forbid logging into broken providers, it's technically possible, that people might make it. We should still leave a reminder in their inbox, that we consider the provider broken. If they use a provider successfully, they can still open an issue in this repository to reopen the provider.

Reply:
But yes, requiring markdown when BROKEN is still a good idea.

Reply:
i would be fine with that, it was just that the existing after_login_hint is a bit weird, pointing the user to things he has already done and i had no good idea what to say instead,

@missytake feel free to add a commit to this pr and fix the text and the condition.